### ISSUE TITLE
[Question] Can't find snippet file

### ISSUE THREAD

**Issue by azzamsa** [AndreaCrotti/yasnippet-snippets#269]
I am using python snippet. `__init__` key is awesome, cause it can autogenerate `self.foo = foo` assigment for me.

But why I can't find this file ?

In `/snippets/python-mode` there are 

1) init_docstring

```
# key: id
# group : definitions
```

2) init

```
# key: init
# group : definitions
```

I dont find this file, here.

```
# key: _init
# group: Special methods
```

But I do find it in other [fork](https://github.com/sten0/yasnippet-snippets/blob/34631b76156501bd771598b061f9612af8e61a46/snippets/python-mode/__init__)

How it can be possible to generate and call `__init__` when there is no the `key` and file ?

I can access it using menu-bar, under special method menu. But where is the actual file located ?

Thanks lot.

**Comment by npostavs** [comment]
Try `M-x yas-visit-snippet-file RET __init__ with assignment RET` (you should be able to `TAB` complete the snippet name).

**Comment by azzamsa** [comment]
@npostavs file does not exists

>__init__ with assignment

```
yasnippet-snippets 20180408.446
```

**Comment by npostavs** [comment]
> How can I use the snippet that [...] comes from /jorgenschaefer/elpy/tree/master/snippets/python-mode

If you installed elpy, I guess it installs those snippets for you.

> Why these snippets separated ?

Because they haven't been merged yet: #259

**Comment by azzamsa** [comment]
> If you installed elpy, I guess it installs those snippets for you.

Sure. My paragraph above just to explain the 'magic' I wondering before.

> Because they haven't been merged yet: #259

Haha. I think because the snippets are framwork-related. Instead I think those snippet in elpy are valid cpython standard.

Thanks lot for the help @npostavs . Thanks for revealing the magic :)
